lib/sle2docker/cli.rb

```diff
@@ -47,7 +47,7 @@ def version
     private
 
     def ensure_can_access_dockerd
-      output = `docker info`
+      output = `docker info 2>/dev/null`
```

Comment:
Ok, let's keep it this way. The only case for failure is that Docker doesn't work, for which we already know what to do/debug.